### DISCUSSION TITLE
"test/go.mod" not detected

### DISCUSSION THREAD

**Discussion by kamatama41** [renovatebot/renovate#8950] [Request Help] [answered]
I'm using `regexManagers` to update `foo/go.mod` and `test/go.mod` file, but the following renovate.json detects only `foo/go.mod`.  Renovate doesn't detect test/go.mod even if fileMatch is `test/go\\.mod`. How can I update the test/go.mod file?

```
.
├── foo
│   └── go.mod
├── renovate.json
└── test
    └── go.mod

```
```json
{
  "extends": [
    "config:base"
  ],
  "regexManagers": [
    {
      "fileMatch": [
        "go\\.mod"
      ],
      "matchStrings": [
        "\ngo (?<currentValue>[0-9.]+)\n"
      ],
      "datasourceTemplate": "docker",
      "depNameTemplate": "golang",
      "versioningTemplate": "docker"
    }
  ]
}
```

Part of dashboard log 

```
DEBUG: Matched 1 file(s) for manager regex: foo/go.mod
```

**Comment by rarkins** [comment, marked as answer]
Test folders are ignored if you are extending the config:base preset. You'll need to override ignorePaths if so

**Reply by kamatama41** [reply]
Ah, got it. Thanks.
